addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -549,7 +549,9 @@ QUnit.module("Fields", (hooks) => {
             assert.strictEqual(input.value, expected);
         }
 
+        await editValue("0", "0");
         await editValue("2", "2.00");
+        await editValue("0.0", "0");
```

Comment:
This is for the integer part below. Here, the expected value should be "0.00". After that, the tests will be okay and I'm okay with this PR.